app/src/main/java/com/asfoundation/wallet/ui/iab/PaymentMethodsAnalytics.kt

```diff
@@ -34,6 +34,7 @@ class PaymentMethodsAnalytics @Inject constructor(
     const val PAYMENT_METHOD_APPC = "appc_c"
     const val PAYMENT_METHOD_LOCAL = "local"
     const val PAYMENT_METHOD_SANDBOX = "sandbox"
+    const val PAYMENT_METHOD_GOOGLEPAY_WEB = "googlepay"
```

Comment:
I think we should have pattern here, use underline or join every word

Reply:
This is the specific key for google pay, we shouldn't define a different one in the app.